guide/qip/qip-basics.rst

```diff
@@ -284,36 +288,54 @@ In addition to these pre-defined gates, QuTiP also allows the user to define the
 Plotting a Quantum Circuit
 ===================================
 
-A quantum circuit (described above) can directly be plotted using the QCircuit library ( https://github.com/CQuIC/qcircuit ).
-QCiruit is a quantum circuit drawing application and is implemented directly into QuTiP. As QCircuit uses LaTex for plotting you need to have several tools installed to use the plotting function within QuTip:
-*pdflatex*, *pdfcrop* and *imagemagick (to convert pdf to png)*.
+A quantum circuit (described above) can directly be plotted using the QCircuit library (https://github.com/CQuIC/qcircuit).
+QCiruit is a quantum circuit drawing application and is implemented directly into QuTiP.
+
+The circuit image visualization requires LaTeX and ImageMagick for display.
+The module automatically generates the LaTeX code for plotting the circuit,
+produces the pdf and converts it to the png format. On Mac and Linux,
+ImageMagick can be easily installed with the command conda install imagemagick if you have conda installed.
+Otherwise, please follow the installation instructions on the ImageMagick documentation.
+
+On windows, you need to download and install ImageMagick installer.
+In addition, you also need perl (for `pdfcrop`) and
+Ghostscript (additional dependency of ImageMagick for png conversion).
```

Comment:
does imagemagick really depend on `gs` for png?? that seems crazy to me - `gs` is PS/PDF normally, and imagemagick is the raster graphics tools

*edit*: no wait, I get it - it needs `gs` to read the pdf

Reply:
also, double back-ticks for `pdfcrop`